test/server/cards/02_SHD/units/SupercommandoSquad.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Supercommando Squad', function() {
+    integration(function() {
+        describe('Supercommando Squad\' ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'supercommando-squad', upgrades: ['shield'] }],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'jedha-agitator'],
+                    }
+                });
+            });
+
+            it('should give it sentinel only as long as it is upgraded', function () {
+                this.player1.pass();
+                this.player2.clickCard(this.wampa);
+                //Supercommando Squad automatically selected due to sentinel
+                expect(this.player1).toBeActivePlayer();
+                //no damage because of shield
+                expect(this.supercommandoSquad.damage).toBe(0);
```

Comment:
Do you mind adding a little bit of whitespace between code blocks for readability?